.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,231 @@
+name: "pull-request"
+
+on:
+  push:
+    branches: [master, develop]
+  pull_request:
+    branches: [master, develop]
+
+env:
+  APP_CONFIG_ECM_HOST: ${{ secrets.PIPELINE_ENV_URL }}
+  ADMIN_EMAIL: ${{ secrets.PIPELINE_ADMIN_USERNAME }}
+  ADMIN_PASSWORD: ${{ secrets.PIPELINE_ADMIN_PASSWORD }}
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  AWS_REGION: ${{ secrets.AWS_DEFAULT_REGION }}
+  CONTENT_CE_DIST_PATH: "./dist/content-ce"
+  APP_CONFIG_PROVIDER: ECM
+  APP_CONFIG_AUTH_TYPE: BASIC
+  APP_CONFIG_OAUTH2_HOST: http://localhost:4200/auth/realms/alfresco
+  PLAYWRIGHT_E2E_HOST: "http://localhost:4200"
+  APP_CONFIG_OAUTH2_CLIENTID: alfresco
+  APP_CONFIG_PLUGIN_AOS: true
+  APP_CONFIG_PLUGIN_CONTENT_SERVICE: true
+  APP_CONFIG_PLUGIN_FOLDER_RULES: true
+  APP_CONFIG_ENABLE_MOBILE_APP_SWITCH: true
+  APP_CONFIG_SESSION_TIME_FOR_OPEN_APP_DIALOG_DISPLAY_IN_HOURS: "12"
+  APP_CONFIG_OAUTH2_IMPLICIT_FLOW: true
+  APP_CONFIG_OAUTH2_SILENT_LOGIN: true
+  APP_CONFIG_OAUTH2_REDIRECT_LOGOUT: /
+  APP_CONFIG_OAUTH2_REDIRECT_LOGIN: /
+  APP_CONFIG_OAUTH2_REDIRECT_SILENT_IFRAME_URI: "{protocol}//{hostname}{:port}/assets/silent-refresh.html"
+
+jobs:
+  setup:
+    name: 'setup'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ secrets.AWS_DEFAULT_REGION }}
```

Comment:
this job is not doing anything useful, aws auth is working everywhere else because you are injecting workflow-wide aws credentials as environment variables and aws cli is automatically picking up them.

remove this job.

Reply:
Right, removed